etl/steps/data/garden/tuberculosis/2023-11-27/drug_resistance_surveillance.meta.yml

```diff
@@ -0,0 +1,236 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Tuberculosis
+        - Causes of Death
+  display:
+    numDecimalPlaces: 0
+tables:
+  drug_resistance_surveillance:
+    variables:
+      pulm_labconf_new:
+        title: New bacteriologically confirmed pulmonary TB cases
+        unit: confirmed cases
+        description_short: Number of new bacteriologically confirmed pulmonary TB cases.
+        description_from_producer: Number of new bacteriologically confirmed pulmonary TB cases.
+      pulm_labconf_ret:
+        title: Previously treated bacteriologically confirmed pulmonary TB cases.
+        description_short: Number of individuals with previously treated for TB that newly have bacteriologically confirmed pulmonary TB.
```

Comment:
For me it's not clear enough. I would change it to something like
```
Number of individuals previously treated for TB that have bacteriologically confirmed pulmonary TB.
```

Reply:
Yeh that's nicer for sure, thanks!